ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.16-lp-interop.yaml

```
@@ -0,0 +1,84 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  mtc-apps-deployer:
+    name: mtc-apps-deployer
+    namespace: mtc-qe
+    tag: master
+  mtc-python-client:
+    name: mtc-python-client
+    namespace: mtc-qe
+    tag: master
+  openshift-cli-installer:
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    mtc-apps-deployer:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-apps-deployer.tar.gz
+    mtc-python-client:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-python-client.tar.gz
+  to: mtc-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name=mtc-aws-ipi-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.14;stream=stable;worker-flavor=m5.xlarge;timeout=2h
```

```suggestion
      CLUSTER1_CONFIG: name=mtc-aws-ipi-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.15;stream=stable;worker-flavor=m5.xlarge;timeout=2h
```

Like you said in your Slack message, this should be 4.15 now